scipy/optimize/_direct.py

```diff
@@ -140,11 +140,11 @@ def direct(
     into hyperrectangles, each having exactly one of the sampling points as
     its center. In each iteration, DIRECT chooses, using the epsilon parameter,
     which defaults to 1e-4, some of the existing hyperrectangles to be further
-    divided. This division process continues until the maximum iterations or maximum
-    function evaluations allowed are exceeded, or the function value is within
-    the desired percentage error of the global minimum (if known). The improved
-    version of the DIRECT algorithm [2]_ is biased towards local search making it
-    effective for functions without too many local minima.
+    divided. This division process continues until the maximum iterations or
+    maximum function evaluations allowed are exceeded, or the function value
+    is within the desired percentage error of the global minimum (if known).
+    The improved version of the DIRECT algorithm [2]_ is biased towards local
```

Comment:
I would not call the locally biased version "improved". It simply converges faster for cases with only a few global minima.

Suggestion: 
The locally biased variant of DIRECT (originally called DIRECT_L) is used by default. It makes the search more locally biased and more efficient for cases with only a few local minima.